examples/gcp/main.tf

```diff
@@ -1,93 +1,90 @@
-      - cluster:
-          certificate-authority-data: >-
-            <certificate-authority-data>
-          server: "https://gardener.garden.canary.k8s.ondemand.com"
-        name: garden
-    users:
-      - user:
-          token: >-
-            <token>
-        name: robot
-    contexts:
-      - context:
-          cluster: garden
-          user: robot
-          namespace: garden-<profile>
-        name: garden-<profile>-robot
-    current-context: garden-<profile>-robot
-
-    EOT*/
-}
-
-resource "gardener_shoot" "test_cluster" {
-  metadata {
-    name      = "test-cluster"
-    namespace = "garden-berlin"
-  }
-  spec {
-    cloud_profile_name = "gcp"
-    secret_binding_name = "<secret_binding>"
-    region  = "europe-west3"
-
-    maintenance {
-      auto_update{
-        kubernetes_version = true
-        machine_image_version = true
-      }
-
-      time_window {
-        begin = "030000+0000"
-        end = "040000+0000"
-      }
-    }
-    networking{
-      type = "calico"
-      nodes ="10.250.0.0/16"
-    }
-
-    provider {
-      type = "gcp"
-      control_plane_config {
-        gcp {
-          zone = "europe-west3-a"
-        }
-
-      }
-      infrastructure_config {
-        gcp {
-          networks {
-            workers = "10.250.0.0/16"
-          }
-        }
-      }
-      worker {
-        max_surge = 1
-        max_unavailable = 0
-        maximum = 2
-        minimum = 2
-        volume {
-          size = "50Gi"
-          type = "pd-standard"
-        }
-        name = "cpu-worker"
-        machine {
-          image  {
-            name = "coreos"
-            version = "2303.3.0"
-          }
-          type = "n1-standard-4"
-        }
-        zones = ["europe-west3-a"]
-      }
-    }
-
-    kubernetes {
-      version = "1.17.2"
-    }
-  }
-}
+provider "gardener" {
+  kube_file = "${file("<my-gardener-service-account-kubeconfig>")}"
+  /*kube_file          =<<-EOT
+    kind: Config
+    clusters:
+      - cluster:
+          certificate-authority-data: >-
+            <certificate-authority-data>
+          server: "https://gardener.garden.canary.k8s.ondemand.com"
+        name: garden
+    users:
+      - user:
+          token: >-
+            <token>
+        name: robot
+    contexts:
+      - context:
+          cluster: garden
+          user: robot
+          namespace: garden-<profile>
+        name: garden-<profile>-robot
+    current-context: garden-<profile>-robot
+
+    EOT*/
+}
+
+resource "gardener_shoot" "test_cluster" {
+  metadata {
+    name      = "test-cluster"
+    namespace = "garden-<profile>"
+  }
+
+  spec {
+    cloud_profile_name  = "gcp"
+    region              = "europe-west3"
+    secret_binding_name = "<secret_binding>"
+
+    networking {
+      nodes    = "10.250.0.0/19"
+      pods     = "100.96.0.0/11"
+      services = "100.64.0.0/13"
+      type     = "calico"
+    }
+
+    provider {
+      type = "gcp"
+
+      infrastructure_config {
+        aws {
```

Comment:
I think I missed this before :(. aws wont work with provider type `gcp`

Reply:
This is the `GCP` example, not the `aws` one

Reply:
yeah `        aws {` under infrastructure config will not work here since this is gcp file.

Reply:
> yeah ` aws {` under infrastructure config will not work here since this is gcp file.

Fixed :)